### PR TITLE
Audio/Video seeking support

### PR DESCRIPTION
**Pull request by joselopez10**
Changes in the omx audio decoder class to support seeking in AAC pipe, plus improvements for the omx video decoder class for cleaner jumps.

### CODE REVIEW

File: omx/gstomxaudiodec.c [hunk 1]
```diff
@@ -403,17 +402,17 @@ gst_omx_audio_dec_loop (GstOMXAudioDec * self)
...
     gst_audio_info_set_format (&self->info,
         gst_audio_format_build_integer (self->pcm_param.eNumData ==
             OMX_NumericalDataSigned,
             self->pcm_param.eEndian ==
             OMX_EndianLittle ? G_LITTLE_ENDIAN : G_BIG_ENDIAN,
             self->pcm_param.nBitPerSample, self->pcm_param.nBitPerSample),
-            self->pcm_param.nSamplingRate, self->pcm_param.nChannels,
-            self->position);
+        self->pcm_param.nSamplingRate, self->pcm_param.nChannels,
+        self->position);
```
Comment (cgomezv) [inline omx/gstomxaudiodec.c:416]: Is this indentation correct? It doesn't look aligned with the previous lines.

File: omx/gstomxaudiodec.c [hunk 2]
```diff
@@ -664,15 +663,8 @@ gst_omx_audio_dec_loop (GstOMXAudioDec * self)
...
       self->started = FALSE;
     } else if (flow_ret == GST_FLOW_FLUSHING) {
-      GST_DEBUG_OBJECT (self, "Flushing -- stopping task");
-      g_mutex_lock (&self->drain_lock);
-      if (self->draining) {
-        self->draining = FALSE;
-        g_cond_broadcast (&self->drain_cond);
-      }
-      gst_pad_pause_task (GST_AUDIO_DECODER_SRC_PAD (self));
+      GST_DEBUG_OBJECT (self, "Flushing ..");
```
Comment (cgomezv) [inline omx/gstomxaudiodec.c]: We could remove the two points, or at least add the third point.

**Review by cgomezv — APPROVED** [review]

File: omx/gstomxaudiodec.c (hunk 1 above)
Comment (joselopez10) [reply omx/gstomxaudiodec.c:416]: Yes, I know its odd. The GNU indentation tool didn't allow me to push until I made that change :S

File: omx/gstomxaudiodec.c (hunk 2 above)
Comment (joselopez10) [reply omx/gstomxaudiodec.c]: Got it. 66ebc2515adec0b1de7ca6ee7d60380af59d1fad

**Pull request merged**